src/views/pages/name/options-with-or-without-welsh.njk

```diff
@@ -0,0 +1,100 @@
+{% set nameEndingOptionsList = 
```

Comment:
Kept this as an include, to reduce the amount of code in the `name.njk` template.